coderd/database/migrations/000256_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE workspace_agent_scripts ADD COLUMN id           uuid UNIQUE NOT NULL DEFAULT gen_random_uuid();
+ALTER TABLE workspace_agent_scripts ADD COLUMN display_name text        NOT NULL DEFAULT '';
+
+CREATE TYPE workspace_agent_script_timing_stage AS ENUM (
+    'start',
+    'stop',
+    'cron'
+    );
+
+CREATE TABLE workspace_agent_script_timings
```

Comment:
I think calling this timings could work but I feel it's primarily a `exec_status` or `exec_result` table, it contains timing information which feels like more of a side-effect.

Renaming it would give us more flexibility to store more data in it if needed.

Reply:
Renaming a table isn't too difficult; we could do this as a follow-up if needed?